src/SourceGenerators/XamlGenerationTests.Core/TestAttachedPropertyOwner.cs

```diff
@@ -60,7 +57,16 @@ public static void SetMoreContent(DependencyObject obj, object value)
 		public static readonly DependencyProperty MoreContentProperty =
 			DependencyProperty.RegisterAttached("MoreContent", typeof(object), typeof(TestAttachedPropertyOwner), new PropertyMetadata(null));
 
+		public static int? GetNullableType(DependencyObject obj)
```

Comment:
Use expression body where possible.
```suggestion
		public static int? GetNullableType(DependencyObject obj) => (int?)obj.GetValue(NullableTypeProperty);
```

Reply:
I also prefer but I respected the syntax in the file.
OK, I replaced my new line only by an expression body to keep the PR readable.